pkg/file/file.go

```diff
@@ -0,0 +1,72 @@
+package file
+
+import (
+	"fmt"
+	"github.com/go-git/go-billy/v5"
+	"github.com/go-git/go-git/v5"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var regexLastIdFinder = regexp.MustCompile("id:([0-9]+)")
+
+type RuleFile struct {
+	Path string
+	File billy.File
+}
+
+func (f *RuleFile) Open(worktree *git.Worktree) (billy.File, error) {
+	file, err := worktree.Filesystem.OpenFile(f.Path, os.O_RDWR, os.ModePerm)
+	if err != nil {
+		return nil, fmt.Errorf("cannot open file %s: %w", f.Path, err)
+	}
+	f.File = file
+	return file, nil
+}
+
+func (f *RuleFile) Get() (string, error) {
+	buffer := new(strings.Builder)
+	_, err := io.Copy(buffer, f.File)
+	if err != nil {
+		return "", fmt.Errorf("cannot get content from file %s: %w",f.File.Name(), err)
+	}
+	return buffer.String(), nil
+}
+
+func (f *RuleFile) Write(content string) error {
+	_, err := f.File.Write([]byte(content))
+	if err != nil {
+		return fmt.Errorf("cannot write to file %s: %w", f.File.Name(), err)
+	}
+	return nil
+}
+
+func (f *RuleFile) Close() error {
+	err := f.File.Close()
+	if err != nil {
```

Comment:
Personally, I find the "all in one line" style appealing (applies to all `if`s in this PR). I think it reduces visual clutter.

Can you combine the two lines above into a single one? e.g.

```suggestion
	if err := f.File.Close(); err != nil {
```

Maybe there is a reason why you would not want to do that. Let me know.